snuba/datasets/storages/metrics.py

```diff
@@ -0,0 +1,76 @@
+from snuba.datasets.metrics_processor import MetricsProcessor
+from snuba.clickhouse.columns import (
+    AggregateFunction,
+    Array,
+    Column,
+    ColumnSet,
+    DateTime,
+    Nested,
+    SchemaModifiers,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.schemas.tables import TableSchema, WritableTableSchema
+from snuba.datasets.storage import ReadableTableStorage, WritableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.table_storage import build_kafka_stream_loader_from_settings
+
+buckets_columns = ColumnSet(
+    [
+        Column("org_id", UInt(64)),
+        Column("project_id", UInt(64)),
+        Column("metric_id", UInt(64)),
+        Column("metric_type", String()),
+        Column("timestamp", DateTime()),
+        Column("tags", Nested([Column("key", UInt(64)), Column("value", UInt(64))])),
+        Column("set_values", Array(UInt(64))),
+        Column("materialization_version", UInt(8)),
+        Column("retention_days", UInt(16)),
+        Column("partition", UInt(16)),
+        Column("offset", UInt(64)),
+    ]
+)
+
+buckets_storage = WritableTableStorage(
+    storage_key=StorageKey.METRICS_BUCKETS,
+    storage_set_key=StorageSetKey.METRICS,
+    schema=WritableTableSchema(
+        columns=buckets_columns,
+        local_table_name="metrics_buckets_local",
+        dist_table_name="metrics_buckets_dist",
+        storage_set_key=StorageSetKey.METRICS,
+    ),
+    query_processors=[],
+    stream_loader=build_kafka_stream_loader_from_settings(
+        StorageKey.METRICS_BUCKETS,
+        processor=MetricsProcessor(),
+        default_topic_name="ingest-metrics",
```

Comment:
Just a reminder this has changed recently. You'll need to add this to the topic enum.

Reply:
Done